test/chess/store/move_test.exs

```diff
@@ -0,0 +1,98 @@
+defmodule Chess.Store.MoveTest do
+  @moduledoc false
+
+  use Chess.DataCase
+
+  describe "move" do
+    alias Chess.Repo
+    alias Chess.Board
+    alias Chess.Store.Move
+
+    import Chess.Factory
+
+    test "move is valid with a game, a from, and a to" do
+      user = insert(:user, %{email: "link@hyrule.com"})
```

Comment:
You can omit the map when using ex_machina:
```elixir
insert(:user, email: "zelda@hyrule.com")

insert(:game,
  board: Board.default,
  user_id: user.id,
  opponent_id: opponent.id
)

```
Elixir will coerce it into a keyword list.